infrastructure/environments/dplplat01/sites.yaml

```
@@ -2,7 +2,7 @@
 x-defaults: &default-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2024.25.1"
+  dpl-cms-release: "2024.26.0"
 x-next: &next-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
```

Husk at slette `x-next` blokken

```suggestion
  dpl-cms-release: "2024.26.0"
```

---

👍